### ISSUE TITLE
Runewords - Make Favorites sortable / Option Show Only Favorites

### ISSUE THREAD

**Issue by eccobold** [Michaelangel007/d2_cheat_sheet#7]
Many runewords are junk, some are ok and some are quite desirable. So it would be nice if this list could be filtered a little better.

- Make the List sortable by "Fav"

OR (harder but probably more usefull)

- Split the List in "Fav" and "Non Fav"
- let the two lists be collapsible

**Comment by Michaelangel007** [comment]
I agree the `Fav` and `S Tier` being hardcoded is a little limiting.

It wouldn't be _too_ hard to sort by Favorite.  I've been holding off since I was planning on making it a toggle to clear all favorites from runewords but making it function like all the other runeword headers and be sortable is probably the better way to go.

I've been debating making the `S Tier` column be user selectable.  That way you could customize your own S-Tier list, sorting would work, and favorites would show the rune manifest.

Thanks for the ideas.  I want to mull this over a bit more before adding functionality but I'll post an update once I've out a solution.
